projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -79,9 +79,16 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
     protected readonly renderingStrategy: RenderingStrategy,
     protected readonly svgUtilService: SvgUtilService,
     protected readonly d3Utils: D3UtilService,
-    protected readonly domRenderer: Renderer2
+    protected readonly domRenderer: Renderer2,
+    protected readonly groupId?: string
   ) {}
 
+  public showCrosshair(locationData: MouseLocationData<TData, Series<TData> | Band<TData>>[]): void {
+    const location = locationData[0].location;
```

Comment:
I think in one of the other files, we were potentially making this array empty, which would cause a runtime error. Even if we fix that, we should be defensive here.